README.md

```diff
@@ -1 +1,42 @@
-# Awesome-book-ES6
\ No newline at end of file
+# Awsome-book-using-ES6
+# capstone project
```

Comment:
- [ ] Please always double-check your README file to make sure that you don't include unnecessary information in your file. Your README file should only contain information concerning your project, this way, your project will be considered professional.

Reply:
edited